examples/ecs-firelens/fluent-bit.conf

```diff
@@ -0,0 +1,41 @@
+[SERVICE]
+    Log_Level debug
+
+# input used for stdout application logs
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+# TCP input used for EMF payloads
+[INPUT]
+    Name        tcp
+    Listen      0.0.0.0
+    Port        25888
+    Chunk_Size  32
+    Buffer_Size 64
+    Format      none
+    Tag         emf-${HOSTNAME} # This tag is used by the output plugin to determine the LogStream
+                                # inclding the HOSTNAME is a way to increase the number of LogStreams
+                                # proportional to the number of instances. The maximum throughput on a
+                                # single LogStream is 5 MB/s (max 1 MB at max 5 TPS).
+
+# Output for EMF over TCP -> CloudWatch
+[OUTPUT]
+    Name                cloudwatch
+    Match               emf-*
+    region              us-east-1
+    log_key             log
+    log_group_name      aws-emf-ecs-firelens-example-metrics
+    log_stream_prefix   from-fluent-bit-
+    auto_create_group   true
+    log_format          json/emf
+
+
+# Output for stdout -> CloudWatch
+[OUTPUT]
+    Name cloudwatch
+    Match example-*
+    region us-east-1
+    log_group_name aws-emf-ecs-firelens-example-logs
+    log_stream_prefix from-fluent-bit-
+    auto_create_group true
```

Comment:
Minor/nit: I personally prefer to have customers define the standard out log configuration in the logConfiguration section of the task definition for each container.

Reply:
moved into container-definitions